.github/workflows/pr-check.yml

```diff
@@ -30,6 +30,14 @@ jobs:
           key: ${{ runner.os }}-${{ matrix.java }}-gradle-${{ hashFiles('**/*.gradle.kts') }}
           restore-keys: |
             ${{ runner.os }}-${{ matrix.java }}-gradle-
+      # used by maven-plugin integration tests
+      - name: Set up Maven cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
```

Comment:
Should we stripe this by java version like we do with Gradle?

Reply:
Yeah we should. 🤞 dokka 11 launches soon so we will be get rid of Java 8 build....